2019-08-15 15:01:08,900 ERROR: Exception on /index [GET] [in /home/alexey/PycharmProjects/flask_site/venv/lib/python3.7/site-packages/flask/app.py:1891]
Traceback (most recent call last):
  File "/home/alexey/PycharmProjects/flask_site/venv/lib/python3.7/site-packages/flask/app.py", line 2446, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/alexey/PycharmProjects/flask_site/venv/lib/python3.7/site-packages/flask/app.py", line 1951, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/home/alexey/PycharmProjects/flask_site/venv/lib/python3.7/site-packages/flask/app.py", line 1820, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/home/alexey/PycharmProjects/flask_site/venv/lib/python3.7/site-packages/flask/_compat.py", line 39, in reraise
    raise value
  File "/home/alexey/PycharmProjects/flask_site/venv/lib/python3.7/site-packages/flask/app.py", line 1949, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/alexey/PycharmProjects/flask_site/venv/lib/python3.7/site-packages/flask/app.py", line 1935, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/alexey/PycharmProjects/flask_site/venv/lib/python3.7/site-packages/flask_login/utils.py", line 261, in decorated_view
    return func(*args, **kwargs)
  File "/home/alexey/PycharmProjects/flask_site/app/routes.py", line 42, in index
    prev_url=prev_url)
  File "/home/alexey/PycharmProjects/flask_site/venv/lib/python3.7/site-packages/flask/templating.py", line 140, in render_template
    ctx.app,
  File "/home/alexey/PycharmProjects/flask_site/venv/lib/python3.7/site-packages/flask/templating.py", line 120, in _render
    rv = template.render(context)
  File "/home/alexey/PycharmProjects/flask_site/venv/lib/python3.7/site-packages/jinja2/asyncsupport.py", line 76, in render
    return original_render(self, *args, **kwargs)
  File "/home/alexey/PycharmProjects/flask_site/venv/lib/python3.7/site-packages/jinja2/environment.py", line 1008, in render
    return self.environment.handle_exception(exc_info, True)
  File "/home/alexey/PycharmProjects/flask_site/venv/lib/python3.7/site-packages/jinja2/environment.py", line 780, in handle_exception
    reraise(exc_type, exc_value, tb)
  File "/home/alexey/PycharmProjects/flask_site/venv/lib/python3.7/site-packages/jinja2/_compat.py", line 37, in reraise
    raise value.with_traceback(tb)
  File "/home/alexey/PycharmProjects/flask_site/app/templates/_post.html", line 27, in template
    {% endif %)
  File "/home/alexey/PycharmProjects/flask_site/venv/lib/python3.7/site-packages/jinja2/environment.py", line 1005, in render
    return concat(self.root_render_func(self.new_context(vars)))
  File "/home/alexey/PycharmProjects/flask_site/app/templates/index.html", line 2, in top-level template code
    {% import 'bootstrap/wtf.html' as wtf %}
  File "/home/alexey/PycharmProjects/flask_site/app/templates/base.html", line 1, in top-level template code
    {% extends 'bootstrap/base.html' %}
  File "/home/alexey/PycharmProjects/flask_site/venv/lib/python3.7/site-packages/flask_bootstrap/templates/bootstrap/base.html", line 1, in top-level template code
    {% block doc -%}
  File "/home/alexey/PycharmProjects/flask_site/venv/lib/python3.7/site-packages/flask_bootstrap/templates/bootstrap/base.html", line 4, in block "doc"
    {%- block html %}
  File "/home/alexey/PycharmProjects/flask_site/venv/lib/python3.7/site-packages/flask_bootstrap/templates/bootstrap/base.html", line 20, in block "html"
    {% block body -%}
  File "/home/alexey/PycharmProjects/flask_site/venv/lib/python3.7/site-packages/flask_bootstrap/templates/bootstrap/base.html", line 23, in block "body"
    {% block content -%}
  File "/home/alexey/PycharmProjects/flask_site/app/templates/base.html", line 48, in block "content"
    {% block app_content %}{% endblock %}
  File "/home/alexey/PycharmProjects/flask_site/app/templates/index.html", line 11, in block "app_content"
    {% include '_post.html' %}
  File "/home/alexey/PycharmProjects/flask_site/venv/lib/python3.7/site-packages/jinja2/environment.py", line 780, in handle_exception
    reraise(exc_type, exc_value, tb)
  File "/home/alexey/PycharmProjects/flask_site/venv/lib/python3.7/site-packages/jinja2/_compat.py", line 37, in reraise
    raise value.with_traceback(tb)
  File "/home/alexey/PycharmProjects/flask_site/app/templates/_post.html", line 27, in template
    {% endif %)
  File "/home/alexey/PycharmProjects/flask_site/venv/lib/python3.7/site-packages/jinja2/environment.py", line 497, in _parse
    return Parser(self, source, name, encode_filename(filename)).parse()
  File "/home/alexey/PycharmProjects/flask_site/venv/lib/python3.7/site-packages/jinja2/parser.py", line 901, in parse
    result = nodes.Template(self.subparse(), lineno=1)
  File "/home/alexey/PycharmProjects/flask_site/venv/lib/python3.7/site-packages/jinja2/parser.py", line 888, in subparse
    self.stream.expect('block_end')
  File "/home/alexey/PycharmProjects/flask_site/venv/lib/python3.7/site-packages/jinja2/lexer.py", line 384, in expect
    self.name, self.filename)
jinja2.exceptions.TemplateSyntaxError: expected token 'end of statement block', got '%'
